FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/IRobot.java

```diff
@@ -17,5 +17,9 @@
 	void setGear(SolenoidValue value);
 	MotorOutput getMotorSpeed();
 	void setArmSpeed(double speed, boolean turbo);
-	void setArmSpeed(double speed, boolean turbo, boolean operatorTurbo);
+	void setArmSpeed(double speed, boolean turbo, boolean operatorTurbo);
```

Comment:
In that case, would moving the turbo constants into TeleopController make sense?